openeo_pg_parser_networkx/pg_schema.py

```diff
@@ -241,7 +241,7 @@ def __repr__(self):
 
 
 class TemporalInterval(BaseModel):
-    __root__: list[Union[Year, Date, DateTime, Time]]
+    __root__: list[Union[Year, Date, DateTime, Time, None]]
```

Comment:
We should add a test to make sure this behaves correctly to test_temporal_intervals too!

Reply:
Ok I added a test for an open ended temporal interval!